feconf.py

```diff
@@ -433,8 +433,12 @@ def get_empty_ratings():
 FEEDBACK_THREAD_URL_PREFIX = '/threadhandler'
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_MESSAGE_EMAIL_HANDLER_URL = (
-    '%s/email/feedbackemailhandler' % TASKQUEUE_URL_PREFIX)
+    '%s/email/batchfeedbackemailhandler' % TASKQUEUE_URL_PREFIX)
```

Comment:
batchfeedbackmessageemailhandler, perhaps

Reasoning: the "status change" url indicates the type of thing being sent; this one probably should too.